spec/unit/mutant/integration/rspec_spec.rb

```diff
@@ -115,10 +115,20 @@
   end
 
   before do
-    expect(RSpec::Core::ConfigurationOptions).to receive(:new).with(%w[spec --fail-fast]).and_return(options)
-    expect(RSpec::Core::Runner).to receive(:new).with(options).and_return(runner)
-    expect(RSpec).to receive(:world).with(no_args).and_return(world)
-    allow(Time).to receive(:now).and_return(Time.now)
+    expect(RSpec::Core::ConfigurationOptions).to receive(:new)
+      .with(%w[spec --fail-fast])
+      .and_return(rspec_options)
+
+    expect(RSpec::Core::Runner).to receive(:new)
+      .with(rspec_options)
+      .and_return(rspec_runner)
+
+    expect(RSpec).to receive(:world)
+      .with(no_args)
+      .and_return(world)
+
+    allow(Time).to receive(:now)
```

Comment:
I don't think it asserts that `Time#now` is called with zero arity, but the method doesn't receive any arguments either. If some code actually attempted to pass an argument to the method it would result in a runtime exception.

IMHO, as mentioned above, it's not necessary to specify `with(...)` except in cases where the method could reasonably accept an argument.